### PR TITLE
Add `pipeline.py` along with necessary files to have a working pipeline.

### PR DESCRIPTION
**Pull request by davidgodinez**

### CODE REVIEW

**Comment by davidgodinez** [comment]
This is the major first commit. Please provide any feedback necessary to make this commit meaningful and correct. Thank you

**Comment by kabilar** [comment]
Thanks @davidgodinez.  Please also submit the associated `element-sensory-stimulation` pull request so that we can run and test these scripts (e.g. `pipeline.py` and `ingest.py`).  Or remove the scripts with `element-sensory-stimulation` dependencies.

File: README.md [hunk 1]
```diff
@@ -1 +1,226 @@
-# workflow-sensory-stimulation
\ No newline at end of file
+# DataJoint Workflow-Sensory-Stimulation
+
+Note: This README is still a work in progress.
```
Comment (kabilar) [inline README.md:3]:
```suggestion
Note: This `workflow-sensory-stimulation` and `element-sensory-stimulation` is still a work in progress and has not been officially released.
```

**Comment by kabilar** [comment]
Thanks for your work here.  Closing since we have created an element-visual-stimulus repository.

**Pull request closed without merging**